arangosh/Import/ImportFeature.cpp

```diff
@@ -201,6 +201,11 @@ void ImportFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
                          actionsJoined,
                      new DiscreteValuesParameter<StringParameter>(&_onDuplicateAction, actions));
 
+  options->addOption("--merge-attributes",
+                     "action to merge attributes into one " +
+                         actionsJoined,
+                     new VectorParameter<StringParameter>(&_mergeAttributes));
```

Comment:
We should add a `.setIntroducedIn(30901)` here so that the argument is reported as being available since version 3.9.1.